src/test/test_bitcoin.h

```diff
@@ -59,14 +65,17 @@ struct CConnmanTest {
 
 class PeerLogicValidation;
 struct TestingSetup: public BasicTestingSetup {
-    fs::path pathTemp;
     boost::thread_group threadGroup;
     CConnman* connman;
     CScheduler scheduler;
     std::unique_ptr<PeerLogicValidation> peerLogic;
 
     explicit TestingSetup(const std::string& chainName = CBaseChainParams::MAIN);
     ~TestingSetup();
+
+    private:
+        // Some tests require -datadir to be set.
+        fs::path pathTemp = getPath("tempdir");
```

Comment:
Should be done in the constructor to make sure `m_path_root` is properly initialized?

Reply:
Also, could be const if possible?